### PR TITLE
HoloToolkit Input model: Adding simple test scene for gaze and air tap

### PR DESCRIPTION
**Pull request by NeerajW**
I've heard feedback on forums, comments and MVP program that folks would like simple starter scenes for the input model. 

1. Adding a simple test scene which has very little code and shows how to respond to gaze with IFocusable. Uses a performant custom shader to highlight the object's material. This is done via GazeResponder.cs
2. Added a simple TapResponder.cs which changes the scale for the object when tapped.

Will add something similar for Navigation rotate and Manipulation move next. 

Hope these help!

### CODE REVIEW

File: Assets/HoloToolkit/Input/Shaders/SpecularHighlight.shader [hunk 1]
```diff
@@ -0,0 +1,177 @@
+﻿//
```
Comment (HodgsonSDAS) [inline Assets/HoloToolkit/Input/Shaders/SpecularHighlight.shader]: Should we be adding additional shaders to the toolkit?

Could we just make a new material out of existing shaders?

**Comment by HodgsonSDAS** [comment]
What's missing from InputManagerTest that's in this PR?

**Comment by NeerajW** [comment]
Please read my description :) There is no simple way for folks to know where to start. InputManagerTest is a great end-end test scene with many components and hierarchy's which does not work for everyone.

File: Assets/HoloToolkit/Input/Shaders/SpecularHighlight.shader (hunk 1 above)
Comment (NeerajW) [inline Assets/HoloToolkit/Input/Shaders/SpecularHighlight.shader]: I could not find any shader that was easily letting me do highlights. 
@dbastienMS does the new shader you are adding give me something like this?

**Comment by HodgsonSDAS** [comment]
> The intent is right but we need to help people move to that point and always ask ourselves, is it easier for me to just use the real API or the toolkit.

That's a good point.

File: Assets/HoloToolkit/Input/Shaders/SpecularHighlight.shader [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+// Copyright (C) Microsoft. All rights reserved.
```
Comment (keveleigh) [inline Assets/HoloToolkit/Input/Shaders/SpecularHighlight.shader]: This is missing the line about the MIT License

**Pull request merged**